core/includes/unicode.inc

```diff
@@ -475,6 +475,9 @@ function decode_entities($text) {
  */
 function backdrop_strlen($text) {
   global $multibyte;
+  if (!isset($text)) {
+    return 0;
+  }
```

Comment:
@kiamlaluno given the discussion in the issue, I am wondering if we'd be better off using `is_string()` here (instead of type hinting):

```suggestion
  
```

...and since we are bailing early, we might wanna move this conditional at the beginning of the function, before `global $multibyte;`.

```suggestion
  // Bail early if no string provided (if $text is NULL for example).
  if (!is_string($text)) {
    return 0;
  }

    global $multibyte;
```